src/src/rx_main.cpp

```diff
@@ -418,11 +434,14 @@ bool ICACHE_RAM_ATTR HandleSendTelemetryResponse()
 
     OtaGeneratePacketCrc(&otaPkt);
 
+    SX12XX_Radio_Number_t transmittingRadio = (geminiMode) ? SX12XX_Radio_All : SX12XX_Radio_Default;    
```

Comment:
why the parentheses here around `(geminiMode)`?

Reply:
For emphasis.

... I'll fix it.